crates/sui-core/src/authority_aggregator.rs

```diff
@@ -927,7 +927,7 @@ where
         // When to give up on the attempt entirely.
         timeout_total: Option<Duration>,
         // The behavior that authorities expect to perform, used for logging and error
-        description: &'static str,
+        description: String,
```

Comment:
we can leave it as-is but convert it to `String` when constructing `TooManyIncorrectAuthorities`, to make the PR cleaner

Reply:
The problem is that `SuiError` cannot contain anything with a lifetime, otherwise it fails to properly deserialize